apps/meteor/app/livechat/server/hooks/verificationCheck.ts

```diff
@@ -28,12 +25,27 @@ callbacks.add(
 				break;
 			}
 			case RoomVerificationState.isListeningToOTP: {
+				const bot = await Users.findOneById('rocket.cat');
```

Comment:
We can cache this in-memory so we don't have to make a DB call for every message. Here's a reference for similar pattern -> https://github.com/RocketChat/Rocket.Chat/blob/a19d5d54b2e1872816d9d3111099d48d1abef4ea/apps/meteor/ee/app/livechat-enterprise/server/hooks/afterInquiryQueued.ts#L8